Devscord.DiscordFramework/Services/ChannelsService.cs

```diff
@@ -1,21 +1,54 @@
-﻿using Devscord.DiscordFramework.Commons;
+﻿using System;
+using Devscord.DiscordFramework.Commons;
 using Devscord.DiscordFramework.Integration;
 using Devscord.DiscordFramework.Middlewares.Contexts;
 using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using Devscord.DiscordFramework.Middlewares.Factories;
+using Serilog;
 
 namespace Devscord.DiscordFramework.Services
 {
     public class ChannelsService
     {
+        private readonly Regex exMention = new Regex(@"\d+", RegexOptions.Compiled);
```

Comment:
Z samej nazwy tu nie wiele wynika i z tego co kojarzę to _ brakuje przed nazwą

Reply:
nazwa jest dobra bo to jest ogólne do wzmianek
brakuje `_`

ale w nowej wersji (jest na masterze) https://github.com/Devscord-Team/Watchman/blob/master/Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsPropertyConversionService.cs

mamy dodane wyciąganie wzmianek z komend

Reply:
więc można to usunąć